packages/core/src/Synchronizer.ts

```diff
@@ -328,6 +329,23 @@ export class Synchronizer extends EventEmitter {
         return tree
     }
 
+    async genEpochTree(epoch: number): Promise<SparseMerkleTree> {
```

Comment:
I think we should use `bigint|number` at here b/c sometime we use get data from smart contract by ethers.js and it return big number I think